FIAT/alfeld_sorokina.py

```diff
@@ -51,53 +50,34 @@ def AlfeldSorokinaSpace(ref_el, degree):
 
 
 class AlfeldSorokinaDualSet(dual_set.DualSet):
-    def __init__(self, ref_el, degree, variant=None):
+    def __init__(self, ref_el, degree):
         if degree != 2:
             raise NotImplementedError("Alfeld-Sorokina only defined for degree = 2")
-        if variant is None:
-            variant = "integral"
-        if variant not in {"integral", "point"}:
-            raise ValueError(f"Invalid variant {variant}")
 
         top = ref_el.get_topology()
         sd = ref_el.get_spatial_dimension()
         entity_ids = {dim: {entity: [] for entity in sorted(top[dim])} for dim in sorted(top)}
 
         nodes = []
-        dims = (0, 1) if variant == "point" else (0,)
-        for dim in dims:
+        for dim in sorted(top):
             for entity in sorted(top[dim]):
                 cur = len(nodes)
+
+                dpts = ref_el.make_points(dim, entity, degree-1)
```

Comment:
Is this right generally (div vertices), or just in the degree 2 case?

Reply:
I am guessing that the general case would include div at a P_{k-1} lattice. I would figure out the rest of the high-order DOFs at some other point. For now we may just leave this here.